### PR TITLE
chore: migrate to owlbot

### PR DESCRIPTION
**Pull request by Neenu1995**

### CODE REVIEW

File: samples/snippets/src/test/java/com/example/errorreporting/QuickStartIT.java [hunk 1]
```diff
@@ -27,6 +27,6 @@
   @Test
   public void testQuickStart() throws Exception {
     // Ensure quick start runs without any exception
-    QuickStart.main(new String[]{});
+    QuickStart.main(new String[] {});
```
Comment (minherz) [inline samples/snippets/src/test/java/com/example/errorreporting/QuickStartIT.java:30]: Is it really necessary?

File: pom.xml [hunk 2]
```diff
@@ -63,6 +63,7 @@
         <artifactId>google-cloud-errorreporting</artifactId>
```
Comment (minherz) [inline pom.xml:63]: should artifact id separate "error" from "reporting" with dash?

**Review by minherz — DISMISSED** [review]
There are few locations were the rename of the cloud-errorreporting looks incomplete.

File: pom.xml (hunk 2 above)
Comment (Neenu1995) [reply pom.xml:63]: Historically, this repo has both errorreporting and error-reporting mixed everywhere. The gapic module is named `google-cloud-errorreporting` and the proto and grpc modules have `proto-google-cloud-error-reporting-v1beta1`  and `grpc-google-cloud-error-reporting-v1beta1` respectively. The artifact Ids published on maven central also reflect this.

The purpose of this PR is to migrate the existing repo as-is to owlbot without any breaking changes. Any changes in the module names or artifact IDs will be breaking changes and out of scope for his PR.

File: samples/snippets/src/test/java/com/example/errorreporting/QuickStartIT.java (hunk 1 above)
Comment (Neenu1995) [reply samples/snippets/src/test/java/com/example/errorreporting/QuickStartIT.java:30]: putting this back

File: pom.xml [hunk 3]
```diff
@@ -63,6 +63,7 @@
...
         <version>0.122.5-beta-SNAPSHOT</version><!-- {x-version-update:google-cloud-errorreporting:current} -->
```
[the remark this answers is not in the document]
Comment (Neenu1995) [reply pom.xml:64]: Not for the gapic module. But the other two have been updated.

File: samples/snippets/src/test/java/com/example/errorreporting/QuickStartIT.java (hunk 1 above)
Comment (Neenu1995) [reply samples/snippets/src/test/java/com/example/errorreporting/QuickStartIT.java:30]: Looks like owlbot is putting it back to fix formatting

**Review by lesv — APPROVED** [review]

**Review by simonz130 — APPROVED** [review]

**Pull request merged**